code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm

```diff
@@ -274,6 +324,7 @@
 			speaker_name = H.real_name
 		rendered_message = speech.render_message()
 		for(var/mob/living/L in player_list)
+			if (L == speech.speaker)	continue//echoes are annoying
```

Comment:
One-line if

Reply:
are alright.

Reply:
He's not saying they don't work, he's saying that it was made policy to not accept PRs with one line ifs to spite me personally. Though now that PJB isn't as active anymore you may be able to slip by undetected.

My advice, just hit enter. Not worth the drama of fighting the bureau of standards and practices.